lib/board.rb

```diff
@@ -0,0 +1,76 @@
+class Board
+  attr_accessor :board
+
+  def initialize
```

Comment:
- [ ] you don't really need to write on the `:board` here, since only methods will be taking care of writing/modifying it, so please use `attr_readers` instead ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))